mooncake-store/include/master_service.h

```diff
@@ -183,12 +183,12 @@ class MasterService {
     struct ObjectMetadata {
         std::vector<Replica> replicas;
         size_t size;
+        std::chrono::steady_clock::time_point lease_timeout;
```

Comment:
We should probably give that member a default value. Something like the minimum time point, maybe the Unix epoch (1970-01-01), would make sense.

Reply:
Sure. As per this doc, https://en.cppreference.com/w/cpp/chrono/time_point/time_point, the default value of time_point is zero, i.e. time_since_epoch() is zero.